code/datums/looping_sounds/_looping_sound.dm

```diff
@@ -37,6 +37,12 @@
 	/// Has the looping started yet?
 	var/loop_started = FALSE
 
+	//Let's you make a "loud" sound that "projects." IE you can hear this sound from a further distance away.______qdel_list_wrapper(list/L)
```

Comment:
Lets, not let's
.______qdel_list_wrapper(list/L)
use code doc comments (///) instead of regular ones

Reply:
For multi line, can you just... do what I did?
Or can doc comments only be single line?

Reply:
See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md multi line autodoc are supported.

Reply:
Unresolved. The current commit uses /* instead of /**